algebra/primefield/src/fft/prefetch.rs

```diff
@@ -19,6 +23,11 @@ pub trait Prefetch {
     fn prefetch_write(&self);
 }
 
+#[cfg(feature = "memadvise")]
+pub trait Madvise {
```

Comment:
Madvise is hard for me to parse, looks like Mad-vise, can we come up with a name that's more readable?